contracts/BlastNativeYield.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IBlast, YieldMode as IBlast__YieldMode, GasMode as IBlast__GasMode} from "./interfaces/IBlast.sol";
```

Comment:
you only need to do YieldMode as IBlast__YieldMode and GasMode as IBlast__GasMode when the same contract also imports YieldMode from IERC20Rebasing

Reply:
Initial commit was wrong: https://github.com/LooksRare/contracts-libs/pull/103/commits/f80299b624bb29f3c05c21cf2ed9fbe11b714574
It's fixed here: https://github.com/LooksRare/contracts-libs/pull/103/commits/ed787620d83cc407b00b388163ef8713c86708d5
Test contract is fixed here: https://github.com/LooksRare/contracts-libs/pull/103/commits/c01c63d91178b262000dfd672c436d1c5da3b44c